ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -29,26 +29,41 @@ chain:
       echo "Waiting for cluster to become available"
       oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${KUBEVIRT_CLUSTER_NAME}
       echo "Cluster became available, creating kubeconfig"
-      bin/hypershift create kubeconfig --name=${KUBEVIRT_CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig
+      bin/hypershift create kubeconfig --name=${KUBEVIRT_CLUSTER_NAME} >${SHARED_DIR}/guest_kubeconfig
```

Comment:
we probably should leave this line.

i'm not saying this is a great name, but `nested_kubeconfig` is what's consistent throughout the ci-operator/step-registry/hypershift, beyond just being used in kubevirt. I think there might be some assumptions in places that the ${SHARED_DIR}/nested_kubeconfig file already exists as well.

Reply:
i think we can update all occurrences of `nested` with `guest` in the context of hypershift/kubevirt throughout the repo. Shouldn't we stick with consistent terminology?